sqlitecluster/SQLite.cpp

```diff
@@ -566,18 +566,25 @@ bool SQLite::write(const string& query) {
     }
 
     // This is literally identical to the idempotent version except for the check for _noopUpdateMode.
-    return _writeIdempotent(query);
+    SQResult ignore;
+    return _writeIdempotent(query, ignore);
```

Comment:
Why do we not support the new parameter for `write`?

Reply:
We could, but I didn't see any requirement for it to make this auth flow work https://github.com/Expensify/Auth/pull/13393
May be I am missing something?

Reply:
seems weird to have it in one API call but not the other. It's not a bug, it's just inconsistent.

Reply:
Got it, updated `write()`